core/modules/syncer.js

```diff
@@ -180,7 +180,8 @@ Syncer.prototype.readTiddlerInfo = function() {
 	var self = this,
 		tiddlers = this.getSyncedTiddlers();
 	$tw.utils.each(tiddlers,function(title) {
-		var tiddler = self.wiki.tiddlerExists(title) && self.wiki.getTiddler(title);
+		if(!self.wiki.tiddlerExists(title)) return;
```

Comment:
Just to note that our coding style is to always use braces around the bodies of "if" statements.